cmd/zed/main.go

```diff
@@ -51,9 +51,16 @@ func dialOptsFromFlags(cmd *cobra.Command, token storage.Token) []grpc.DialOptio
 		grpc.WithChainUnaryInterceptor(interceptors...),
 	}
 
+	if cobrautil.MustGetBool(cmd, "insecure") && cobrautil.MustGetString(cmd, "cafile") != "" {
+		panic("cafile flag cannot be combined with insecure")
```

Comment:
This should definitely be checked somewhere else and return an error rather than panicking.

Reply:
I don't seen an obvious place to put a "verify flags" precondition; I'm not that familiar with Cobra

Reply:
Yeah, that's pretty fair -- I think we should just ignore the CA flag if the insecure flag is set. The insecure flag description is "connect over a plaintext connection", so if you specify it, that's what we should do.